### PR TITLE
Fix/giphy search

### PR DESCRIPTION
**Pull request by CrazyLemon**
Pls merge this and ignore the mess with commits... ktnx

### CODE REVIEW

**Comment by CrazyLemon** [comment]
@zdobersek ping

**Comment by zdobersek** [comment]
hai

File: scripts/chatter.coffee [hunk 1]
```diff
@@ -121,12 +121,10 @@ module.exports = (bot) ->
     r.reply bot.random vic
 
   bot.regexp /^\.gif (.+)/, ".gif <query> -- Prikaže naključni gif", (match, r) ->
-      url = "http://giphy.com/search/#{encodeURI(match[1].replace(/[^a-z0-9\s]/g,"").replace(/\s\s+/g, "-"))}"
-      bot.fetchHTML url.replace(/%20/g, '-'), ($) ->
-        count = $("#searchresults .found-count").text().split(" GIFs found for")
-        if count[0] > 0
-          gif_id = $(".hoverable-gif a").eq(0).attr("data-id")
-          r.reply "http://media.giphy.com/media/#{gif_id}/giphy.gif"
+      url = "http://api.giphy.com/v1/gifs/search?q=#{encodeURI(match[1].replace(/[^a-z0-9\s]/g,"").replace(/\s\s+/g, "+"))}&api_key=dc6zaTOxFJmzC"
```
Comment (zdobersek) [inline scripts/chatter.coffee]: /^[a-z0-9\s]/g ?

Comment (zdobersek) [inline scripts/chatter.coffee]: /[^a-zA-Z0-9\s/g ?

**Comment by zdobersek** [comment]
This is still wrong, but because it's you, I'll allow it.

**Pull request merged**